rules/integrations/aws/exfiltration_ec2_snapshot_change_activity.toml

```diff
@@ -23,7 +23,62 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS EC2 Snapshot Activity"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating AWS EC2 Snapshot Activity
+
+Amazon EC2 snapshots are a mechanism to create point-in-time references to data that resides in the storage volume.
+System administrators commonly use this for backup operations and data recovery.
+
+This rule looks for the modification of snapshot attributes using the API `ModifySnapshotAttribute` action. This can be
+used to share the snapshot with unauthorized third parties, which means you are giving others access to all the data on
```

Comment:
```suggestion
used to share snapshot data with unauthorized third parties and give them access. 
```

Reply:
I think it is important to highlight that a snapshot is a exact copy of the disk.